### PR TITLE
[gatesim] initial compile clean

### PR DESCRIPTION
**Pull request by jdonjdon**
Initial check in after compile without SDF.
There will be subsequent update.

### CODE REVIEW

File: hw/top_earlgrey/dv/env/chip_if.sv [hunk 1]
```diff
@@ -566,8 +575,14 @@ interface chip_if;
   };
 
   for (genvar i = 0; i < NUM_PWM_CHANNELS; i++) begin : gen_pwm_if_conn
-    pwm_if pwm_if(.clk  (`CLKMGR_HIER.clocks_o.clk_aon_powerup),
+    pwm_if pwm_if(
+`ifdef GATE_LEVEL
+                  .clk (0),
```
Comment (msfschaffner) [inline hw/top_earlgrey/dv/env/chip_if.sv:580]: Are some of these tied-off paths overridden in the closed-source environment?

File: hw/top_earlgrey/dv/tb/tb.sv [hunk 2]
```diff
@@ -533,4 +538,28 @@ module tb;
...
+     end
+
+`define GSIM_FF_D(FF_) \
+     initial begin \
+       force FF_.D = 0; \
+       @(posedge FF_.CK);\
+       release FF_.D;\
+     end
+  `GSIM_FF_D(tb.dut.top_earlgrey.u_pinmux_aon.u_pinmux_strap_sampling.dft_strap_q_reg_0_)
+  `GSIM_FF_D(tb.dut.top_earlgrey.u_pinmux_aon.u_pinmux_strap_sampling.dft_strap_q_reg_1_)
```
Comment (msfschaffner) [inline hw/top_earlgrey/dv/tb/tb.sv:561]: Does this belong here, or in the closed-source?

Also, why is this needed?
Are the flops not reset correctly in the GLS?

File: hw/top_earlgrey/dv/tb/tb.sv [hunk 3]
```diff
@@ -533,4 +538,28 @@ module tb;
   `include "../autogen/tb__xbar_connect.sv"
   `include "../autogen/tb__alert_handler_connect.sv"
 
+  // Gatesim initial
+  `ifdef GATE_LEVEL
+     initial begin
+       // unconnected ports
+       force tb.dut.u_ast.u_entropy.dev1_entropy_o = 'h0;
```
Comment (msfschaffner) [inline hw/top_earlgrey/dv/tb/tb.sv:545]: Is this also needed in the closed-source, or is this exclusive to open-source modelling?

**Review by msfschaffner — COMMENTED** [review]
Thanks @jdonjdon, just a few clarification questions.

File: hw/top_earlgrey/dv/env/chip_if.sv (hunk 1 above)
Comment (jdonjdon) [reply hw/top_earlgrey/dv/env/chip_if.sv:580]: Yes, more precisely signal name doesn't exist anymore.
These are ok for our pilot test.
Some of them will be reverted back if necessary as we expand our gsim test sets.

File: hw/top_earlgrey/dv/tb/tb.sv (hunk 2 above)
Comment (jdonjdon) [reply hw/top_earlgrey/dv/tb/tb.sv:561]: Partner's latest run script in fact remove unit delay.
So this might be outdated.
Let me add `TODO` here to revisit but leave it as is for temporary.
There will be proper clean up.

File: hw/top_earlgrey/dv/tb/tb.sv (hunk 3 above)
Comment (jdonjdon) [reply hw/top_earlgrey/dv/tb/tb.sv:545]: It was added by partners.
In a nut shell, all these can be hidden with `include if necessary.
But there will more forces (I suspect). So keep it here for temporary and will cleanup once
we have stablized gsim harness.

File: hw/top_earlgrey/dv/env/chip_if.sv (hunk 1 above)
Comment (msfschaffner) [reply hw/top_earlgrey/dv/env/chip_if.sv:580]: OK sounds good.

**Review by msfschaffner — APPROVED** [review]
